lib/modules/translation/widgets/asset.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:andax/modules/translation/screens/translation_editor.dart';
+import 'package:andax/modules/translation/services/assets.dart';
+import 'package:andax/shared/widgets/loading_dialog.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'asset_editor.dart';
+
+class Asset extends StatelessWidget {
+  const Asset(
+    this.id, {
+    this.icon,
+    Key? key,
+  }) : super(key: key);
+
+  final IconData? icon;
+  final String id;
+
+  @override
+  Widget build(BuildContext context) {
+    final editor = context.watch<TranslationEditorState>();
+    final change = editor.changes[id]?.value;
+    final base = editor.base[id];
+    final target = change ?? editor.target[id];
+    return ListTile(
+      leading: icon == null ? null : Icon(icon),
+      title: target == null ? null : Text(target),
+      subtitle: base == null ? null : Text(base),
+      trailing: change == null ? null : const Icon(Icons.edit_rounded),
```

Comment:
I really feel like Dart needs an operator for this, just like they `??`. Imagine if you could write `icon !! Icon(icon)` instead of `icon == null ? null : Icon(icon)`, for example!

Reply:
Yes!!! I'm having this thought constantly. Perhaps you could file an issue in their repo?

Reply:
Yes, I really wanted to, just waiting till I get some free time 😅